test/vfd_swmr_group_writer.c

```diff
@@ -3109,37 +5450,39 @@ main(int argc, char **argv)
      * If the named pipe is not forced to communicate in every step, the reader may go ahead
      * to verify the group and the attribute operations before the writer has a chance to
      * carry out the corresponding operations. */
-    if (s.at_pattern != ' ') {
-        s.attr_test = true;
-        if (s.use_named_pipes)
-            s.csteps = 1;
-    }
+    if (s.attr_test && s.use_named_pipes)
+        s.csteps = 1;
+
+    /* For group operation test, force the named pipe to communicate in every step. */
+    if (s.grp_op_test && s.use_named_pipes)
+        s.csteps = 1;
 
     if (writer) {
         for (step = 0; step < s.nsteps; step++) {
             dbgf(2, "writer: step %d\n", step);
 
-            wg_ret = write_group(&s, step);
+            wg_ret = group_operations(&s, step);
 
             if (wg_ret == false) {
                 H5_FAILED();
                 AT();
                 printf("write_group failed at step %d\n", step);
 
                 /* At communication interval, notifies the reader about the failture and quit */
```

Comment:
Minor: spelling--"failture"--> should be failure

Reply:
Thanks done.